.github/workflows/trivy.yml

```
@@ -2,22 +2,30 @@ name: dsp-appsec-trivy
 on:
   pull_request:
   schedule:
-    - cron: '0 14 * * 1' # each Monday at 9am EST
+    - cron: '0 0 * * 0'
```

Github Actions recommends not choosing midnight or round number hours for cron.

Suggestion runs at 1:23 AM.

```suggestion
    - cron: '23 1 * * 0'
```

---

Just committed the suggestion. Thank you @aednichols!